api-reference/10 UI Widgets/Component/1 Configuration/onOptionChanged.md

```diff
@@ -24,3 +24,122 @@ The modified option if it belongs to the first level. Otherwise, the first-level
 The modified option's new value.
 
 ---
+
+The following example shows how to subscribe to changes for an option of {WidgetName}:
```

Comment:
```suggestion
The following example shows how to subscribe to changes of a **{WidgetName}** option:
```

Reply:
If WidgetName starts with a vowel (e.g. Action Sheet), the part of the sentence is "changes of **a A**ction Sheet option". So is "The following example shows how to subscribe to changes of an option" good?